src/motor_node.cc

```diff
@@ -240,7 +240,7 @@ void initMcbParameters(std::unique_ptr<MotorHardware> &robot )
         ROS_INFO("Wheel slip nulling will be enabled for this 4wd system when velocity remains at zero.");
     }
 
-    // Tell the MCB board what the I2C port on it is set to (mcb cannot read it's own switchs!)
+    // Tell the MCB board what the port that is on the Pi I2c says on it (the mcb cannot read it's own switchs!)
```

Comment:
I still don't quite get what this is about exactly, but I think we can leave it for some other time.